flink-end-to-end-tests/flink-elasticsearch1-test/src/main/java/org/apache/flink/elasticsearch1/test/Elasticsearch1SinkExample.java

```diff
@@ -37,20 +38,33 @@
 import java.util.Map;
 
 /**
- * This example shows how to use the Elasticsearch Sink. Before running it you must ensure that
- * you have a cluster named "elasticsearch" running or change the cluster name in the config map.
+ * End to end test for Elasticsearch1Sink.
+ *
+ * <p>This example shows how to use the Elasticsearch1 Sink from an user endpoint. Before running it you
+ * must ensure that you have a cluster named "elasticsearch" running or change the cluster name in the config map.
```

Comment:
This Javadoc doesn't make sense when considering it as a job simply used in end-to-end tests.

Reply:
Will delete.